secboot/secboot.go

```diff
@@ -255,6 +255,9 @@ type UnlockResult struct {
 	// - UnlockedWithSealedKey
 	// - UnlockedWithKey
 	UnlockMethod UnlockMethod
+	// Keyslot is the name of the keyslot used or the name of the
+	// associated keyfile used.
+	Keyslot string
```

Comment:
What are the possible values here? I see that this is compared with things like "external:legacy-fallback" and might be good to document what can be expected.